src/experiments.js

```diff
@@ -616,3 +616,15 @@ window.Altis.Analytics.onReady( () => {
 // Fire a ready event once userland API has been exported.
 const readyEvent = new CustomEvent( 'altis.experiments.ready' );
 window.dispatchEvent( readyEvent );
+
+document.querySelectorAll( '.ab-test-xb-preview' ).forEach( function ( xb ) {
+	let templates = document.querySelectorAll( 'template[data-parent-id="' + xb.dataset.clientId + '"]' );
```

Comment:
My editor kept throwing an error for this and suggested the use of "let". Should it be a var instead?

Reply:
If you're ever in doubt check caniuse.com, eg. https://caniuse.com/let

In this case `let` is fine as we don't support IE older than 10, and as this is for admin users only we don't even support IE11 there.

I would split this out into a separate JS file though that's only loaded when we're previewing content to keep the end user JS as small and lean as it can be.

Reply:
Oooh that's neat! I always thought https://caniuse.com/ was for CSS, Thank you, I now know that it can be used for front-end languages!

Reply:
For the separate file is there a particular method to adding it and including it in the build process? I have tried to get it to work but feel I may be missing something furhter.

I have tried a `wp_enqueue_script` in `aws-analytics/inc/blocks/ab-test/register.php` no luck. I have also tried following the setup of the other files but no success for me yet.

Reply:
You can push work in progress and I can comment on that or we can jump on a huddle. It might not even strictly need to be built.